users/nazu/rules.mk

```
@@ -0,0 +1,14 @@
+SRC += $(USER_PATH)/nazu.c \
+        $(USER_PATH)/keyrecords/process_records.c \
```

No nazu.c exists, so it's erroring out on that file (well, "nazu.o" specifically). 
```suggestion
SRC += $(USER_PATH)/keyrecords/process_records.c \
```

---

You are totally right... that's what happens when blindly copying code.